src/components/calculator.js

```diff
@@ -1,46 +1,56 @@
+          {operation}
+          {next}
+        </span>
         <div className="stylebuttoms">
           <div className="styleCalculator">
-            <button type="button">AC</button>
-            <button type="button">7</button>
-            <button type="button">4</button>
-            <button type="button">1</button>
+            <button type="button" onClick={this.valbutton}>AC</button>
+            <button type="button" onClick={this.valbutton}>7</button>
+            <button type="button" onClick={this.valbutton}>4</button>
+            <button type="button" onClick={this.valbutton}>1</button>
           </div>
           <div className="styleCalculator">
-            <button type="button">+/-</button>
-            <button type="button">8</button>
-            <button type="button">5</button>
-            <button type="button">2</button>
+            <button type="button" onClick={this.valbutton}>+/-</button>
+            <button type="button" onClick={this.valbutton}>8</button>
+            <button type="button" onClick={this.valbutton}>5</button>
+            <button type="button" onClick={this.valbutton}>2</button>
           </div>
           <div className="styleCalculator">
-            <button type="button">%</button>
-            <button type="button">9</button>
-            <button type="button">6</button>
-            <button type="button">3</button>
+            <button type="button" onClick={this.valbutton}>%</button>
+            <button type="button" onClick={this.valbutton}>9</button>
+            <button type="button" onClick={this.valbutton}>6</button>
+            <button type="button" onClick={this.valbutton}>3</button>
           </div>
           <div className="styleCalculator">
-            <button className="buttonfunction" type="button">÷</button>
-            <button className="buttonfunction" type="button">*</button>
-            <button className="buttonfunction" type="button">-</button>
-            <button className="buttonfunction" type="button">+</button>
-
+            <button className="buttonfunction" type="button" onClick={this.valbutton}>÷</button>
+            <button className="buttonfunction" type="button" onClick={this.valbutton}>*</button>
+            <button className="buttonfunction" type="button" onClick={this.valbutton}>-</button>
+            <button className="buttonfunction" type="button" onClick={this.valbutton}>+</button>
           </div>
         </div>
         <div className="stylebottomcalculator">
-          <button className="buttonZero" type="button">0</button>
-          <button type="button">.</button>
-          <button className="buttonfunction" type="button">=</button>
+          <button className="buttonZero" type="button" onClick={this.valbutton}>0</button>
+          <button type="button" onClick={this.valbutton}>.</button>
+          <button className="buttonfunction" type="button" onClick={this.valbutton}>=</button>
         </div>
       </div>
     );
```

Comment:
- [ **OPTIONAL** ] **In README.md File**: I think it would be better if you improve your project's description by using the one(description) provided by your **[sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/sneak_peek.md)** to have a meaningful detail about your project.

- [ **OPTIONAL** ] **In README.md File:** I also suggest you update the content of your `## Getting Started` section of your README file by making it professional. It contains some typing errors(e.g. repository name is `Math-magicians` and not ` To Do List`). I suggest you use **[this template](https://github.com/tresorsawasawa/math_magicians/blob/development/README.md)** to make your README more professional

Reply:
ok, I change the README.md File